### PR TITLE
Checking alert coditions flags to differentiate alert from notifications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
alerting message have three flags in payload to confirm the message
is alerting message or not

flags are :
  alert_condition_status
  alert_condition_state
  alert_condition_unset

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n please review

**Comment by GowthamShanmugam** [comment]
Modified as per @brainfunked comment

File: tendrl/node_agent/message/handler.py [hunk 1]
```diff
@@ -37,12 +37,21 @@ def read_socket(self, sock, *args):
             msg = struct.unpack(frmt, data)
             message = Message.from_json(msg[0])
             # Logger is in commons so passing alert from here
-            notify_only = message.payload.get('notify_only_ui', False)
-            if message.priority == NOTICE_PRIORITY and not notify_only:
```
Comment (r0h4n) [inline tendrl/node_agent/message/handler.py]: @shtripat since notify_only_ui is removed. Make these changes to job status alerts too in commons

Comment (shtripat) [reply tendrl/node_agent/message/handler.py]: @r0h4n sent https://github.com/Tendrl/commons/pull/728 for this.
Also gluster-integration `sds_sync/event_utils.py` need changes. @GowthamShanmugam do the needful as discussed and verify the scenario volume stop and start and make sure alerts are created and cleared respectively in this case.

Comment (GowthamShanmugam) [reply tendrl/node_agent/message/handler.py]: @shtripat @r0h4n i am doing the changes in sds_sync/event_utils.py

**Pull request merged**